.github/workflows/test.yaml

```
@@ -63,4 +66,20 @@ jobs:
 
       - run: patchright install --no-shell chromium
 
-      - run: pytest tests/${{ matrix.test }}.py
+      - name: Install Playwright and Chromium
+        run: |
+          uv pip install playwright
+          uv run playwright install chromium
+          uv run patchright install chrome
+
+      - name: Launch Chrome with remote debugging
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y google-chrome-stable
+          google-chrome --headless --disable-gpu --remote-debugging-port=9222 --no-sandbox &
+
+          # Give Chrome a few seconds to start
+          sleep 5
+
+      - name: Run tests
+        run: uv run --with=dotenv pytest tests/${{ matrix.test }}.py
```

```suggestion
      - run: pytest tests/${{ matrix.test }}.py
```
leave this line as-is